include/net/if.h

```diff
@@ -208,6 +208,12 @@ enum can_ioctl_state_e
   CAN_STATE_BUSY             /* The can/lin bus is busy */
 };
 
+struct can_ioctl_state_s
+{
+  uintptr_t priv;             /* This is private data. */
```

Comment:
where use it ?

Reply:
Because the car's autosar stipulates that there are two modes of sleep and wake-up, internal/noraml, so a flag bit needs to be added to distinguish internal/noraml. The reason why priv is selected here is also for convenience and can be used for other uses later. Increase practicality

Reply:
lower half driver